jsonnet/patch-rules.libsonnet

```diff
@@ -74,6 +81,37 @@ local patchedRules = [
       },
     ],
   },
+  {
+    name: 'thanos-query.rules',
```

Comment:
For convenience purpose, would it be meaningful to update the script to be able to apply changes to all the rules of a group? In your case, applying the same duration and severity to the whole group without having to specify the name of the rules.

Currently, this can be hack around for all the alerting rules of a group by setting the alert name to `""`.

Reply:
yes it would make sense. Will think about it :)